cmd/pillar/server.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
+	logrus "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	cloud "github.com/mattermost/mattermost-cloud/model"
+
+	"github.com/mattermost/pillar/api"
+	"github.com/mattermost/pillar/utils"
+)
+
+const defaultLocalServerAPI = "http://localhost:8078"
+
+var instanceID string
+
+func init() {
+	viper.SetEnvPrefix("PILLAR")
+	viper.AutomaticEnv()
+
+	instanceID = utils.NewID()
+
+	// Core Settings
+	serverCmd.PersistentFlags().String("listen", ":8078", "The interface and port on which to listen on the API.")
+	serverCmd.PersistentFlags().Bool("debug", false, "Whether to output debug logs.")
+
+	// Dev Settings
+	serverCmd.PersistentFlags().Bool("dev", false, "Set to run in dev mode.")
+
+	// Provisioner Settings
+	serverCmd.PersistentFlags().String("cloud-url", viper.GetString("CLOUD_URL"), "Endpoint where the Cloud Provisioning Server can be reached (include the scheme and port number) | ENV: PILLAR_CLOUD_URL")
+}
+
+// Config holds the configuration for pillar.
+type Config struct {
+	DevMode   bool
+	DebugLogs bool
+	CloudURL  string
+}
+
+var serverCmd = &cobra.Command{
+	Use:   "server",
+	Short: "Run the customer web server.",
```

Comment:
```suggestion
	Short: "Run Pillar API server.",
```
I guess should be something like that 😄